locale/pt_BR/emailTemplates.xml

```diff
@@ -16,30 +16,32 @@
 		<subject>Solicitação de atualização de senha</subject>
```

Comment:
Maybe to create another PR for this?!
p.s.: I'm not sure I should spend time with locales, but I'm ok to review the rest of the text.

Reply:
Typically, we only modify `en_US` during development. That's because most languages have dedicated translators who are approved and managed by @mtub. I'm not sure who that is for `pt_BR`, but in general I'd suggest leaving it off.

If you want, you can create a separate PR, but it would be approved by @mtub.

Reply:
Thanks, so I'll leave only the changes connected with the task, the rest I'll re-check against /master and create another PR.